JSDemos/Demos/Form/Validation/description.md

```diff
@@ -1,10 +1,36 @@
-﻿This demo shows how to&nbsp;validate the [Form][0] editors. [Validation rules][1] are specified for each form item using the **validationRules** property. All editors on&nbsp;the form are combined into a&nbsp;validation group defined by&nbsp;the form&rsquo;s [validationGroup][2] property. 
+﻿This demo shows how to validate Form editors. To apply validation rules to an editor, declare them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/) array. Specify [type](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#type) and other properties for each rule. 
 
 
-Note that the _&laquo;Register&raquo;_ button here does not implement a&nbsp;usual [onClick][3] event handler. Instead, it&nbsp;has the [useSubmitBehavior][4] property set to _true_. This setting tells the button to&nbsp;validate and submit the HTML form in&nbsp;which it&nbsp;is&nbsp;nested, with no&nbsp;further configuration required.
+The following validation rules are shown in this demo:  
+- [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/)  
+Demands that a validated editor has a value.
+
+- [CompareRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/)    
+Compares an editor value with a specified expression.
+
+- [PatternRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/PatternRule/)    
+Checks whether an editor value matches a specified pattern.
+
+- [RangeRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/)    
+Checks whether an editor value is in a specified range.
+
+- [StringLengthRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/)  
+Demands that an editor value length is in a specified range.
+
+- [EmailRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/EmailRule/)    
+Demands that an editor value matches the Email pattern.
+
+- [AsyncRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/)    
+Allows you to specify any conditions to be checked on the server. This check is performed last, after all other rules pass validation checks. In this demo, this rule checks whether user input is equal to the `test@dx-email.com` string.
```

Comment:
`the check is performed` - passive and the word `perform` is best to avoid. 

In the end, I changed everything a little bit - just made it a bit more straightforward I guess. 

```suggestion
Allows you to add custom server-side validation logic. Rules of this type run last, only if all other rules passed. In this demo, an AsyncRule checks whether user input matches `test@dx-email.com`.
```